docs/1.0. Certificate Provisioning.md

```diff
@@ -47,67 +47,79 @@ The entity that is providing the EST API in accordance with [RFC 7030][RFC-7030]
-2. When the EST Client is connected to the target environment's network, it will first discover the location of the EST Server using Unicast DNS-SD.
-3. The EST Client should then request the Root CA for the target network, from the EST Server
-    - The EST Client should assume the EST server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
-    - Using the Root CA returned to secure further communications with the EST Server and NMOS Servers
-4. The EST Client must generate a Certificate Signing Request(CSR) for all supported digital signature algorithms with appropriate key lengths
-    - Included in the CSR must be the DNS resolvable name for the current domain of the EST Client
-    - Appropriate values for the remaining CSR fields should be used
-5. The EST Client should send each CSR to the EST Server
-    - The EST Client must provide the manufacturer issued certificate during the TLS handshake, if valid
-    - The EST Server will use the TLS Client Certificate to authenticate and then authorise whether the EST Client. If the EST Client meets the local CA's security policy, it will be issued with a TLS Server Certificate.
-6. The EST Client should provide the returned TLS Certificate and the corresponding chain of trust in all future responses returned by its NMOS APIs
+1. When the NMOS Client (EST Client) is connected to the target environment's network, it will first discover the location of the EST Server using unicast DNS-SD.
+2. The NMOS Client will then request the Root CA for the target network, from the EST Server.
+   - The NMOS Client can assume the EST Server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
+   - The NMOS Client uses the Root CA returned to secure further communications with the EST Server and the NMOS Servers.
 
-The workflow if no valid manufacturer TLS Client Certificate is present or if EST Client is moved to a new network are covered in detail in the following [sections](#est-client).
+The workflow if the NMOS Client is moved to a new network is covered in detail in the [EST Client Behaviour](#est-client-behaviour) section.
 
-![NMOS EST WorkFlow](images/nmos_est_workflow.png)
+![NMOS Client Workflow](images/nmos_client_workflow.png)
+## NMOS Server Automated Certificate Provisioning Flow (informative)
+
+To enable zero-configuration TLS Certificate provisioning, NMOS implementations MUST include a unique TLS Client Certificate, signed by the manufacturer's Certificate Authority. Manufacturers MUST provide the corresponding Root Certificate Authority public key to customers, to be loaded into the EST Server. If the TLS Client Certificate or the chain of trust is compromised, the manufacturer must revoke the comprised certificates.
```

Comment:
```suggestion
To enable zero-configuration TLS Certificate provisioning, NMOS implementations MUST include a unique TLS Client Certificate, signed by the manufacturer's Certificate Authority. Manufacturers MUST provide the corresponding Root Certificate Authority public key to customers, to be loaded into the EST Server. If the TLS Client Certificate or the chain of trust is compromised, the manufacturer MUST revoke the comprised certificates.
```

Reply:
...although this section is marked informative, and is already using "MUST".  This needs fixing.

Reply:
Agreed! Will have a look again.